CONTRIBUTING.md

```diff
@@ -1,6 +1,38 @@
 # Contributing
 First of all, thank you for contributing. It’s appreciated. This guide details how to use issues and pull requests to improve this project.
 
+Table of Contents
+=================
+
+  * [Requirements:](#requirements)
+  * [Installation](#installation)
+  * [Running](#running)
+  * [Making changes](#making-changes)
+    * [Branching](#branching)
+    * [Commit message](#commit-message)
+  * [Creating an Issue](#creating-an-issue)
+  * [Tests](#tests)
+
+## Requirements:
+* NodeJS(v4 LTS)
+* git crypt
+
+## Installation
+Clone this repository
+
+```bash
+$ git clone https://github.com/sphereio/sphere-product-type-import.git
+```
+
+We are using git-crypt for encrypting configs.
+Run
+
+```bash
+$ git-crypt unlock
+```
+
+to unlock the config file.
```

Comment:
This should be used only within our company. External developers shouldn't use this, otherwise we might as well have the credentials hardcoded in plain text.

I would not mention this for now, if people will start asking for it can talk about it.